www/source/partials/docs/_using-hab-continuous-deployment.html.md.erb

```diff
@@ -0,0 +1,56 @@
+# <a name="continuous-deployment" id="continuous-deployment" data-magellan-target="continuous-deployment">Continuous Deployment Using Channels</a>
+
+Continuous deployment is a well-known software development practice of building and testing code changes in preparation for a release to a production environment. 
+
+<%= partial "/partials/global/channel-overview" %>
+
+By default, every new package is placed in the `unstable` channel by Builder.  Packages in the `unstable` channel cannot be started or installed unless you specify the `--channel` flag in the `hab` CLI, or set the `HAB_BLDR_CHANNEL` environment variable to a non-stable channel. This is because the default channel used by the `hab` CLI when starting, installing, or loading packages is the `stable` channel. The `stable` channel indicates a level of stability and functionality suitable for use in multi-service applications or as a dependency for your service.
+
+To promote your package to a channel, you must first upload it to Builder and then use the `hab pkg promote` subcommand to promote the package to the intended channel. To combine operations, the `hab` CLI allows you to do both in one command by using the `--channel` option when uploading your package. The following shows how to upload and promote a package to a custom channel named `test`.
+
+```bash
+$ hab pkg upload -z <OAuth_token> results/<hart file> --channel test
+```
+
+In the example above, if you look up your package in the Builder UI, or using the `hab pkg channels` subcommand, you can see that your package is tagged for both the `test` and `unstable` channels.
+
+> **Note** Custom channels like `test` are scoped to each package. Builder does not create channels scoped to an origin, so if you want to use custom channels for future releases of a package, you must promote to those channels for each release.
+
+If you have already uploaded your package to a channel and wish to promote it to a different channel, use the `hab pkg promote` subcommand as shown below.
+
+```bash
+$ hab pkg promote -z <OAuth_token> <origin>/<package>/<version>/<release> stable
+```
+
+## Combining an Update Strategy with Channels
+
+By using both channels and either the `at-once` or `rolling` update strategies, you can automatically update packages in a given channel as shown below:
```

Comment:
It might be helpful to link to explanations of how these update strategies work.